api/seeds/shared/set-up-users.js

```diff
@@ -34,7 +40,14 @@ const createUsersToAdd = async (knex, oktaClient) => {
     .then(role => role.id);
 
   logger.info('Setting up affiliations to add');
-  const oktaAffiliations = [];
+  const oktaAffiliations = states.map(state => ({
```

Comment:
This makes sense given our datamodel being "State-based" but it does seem strange to add 50 rows to a table to make someone a system admin.

Reply:
We have "federal" as a state.  I did some other work that just didn't append a state filter if the state was "federal".  IDK if that would work or make sense here.